app/services/submissions_inviter.rb

```diff
@@ -5,7 +5,9 @@ def initialize(event_dates, event_venue)
   end
 
   def call(submissions)
-    if submissions.empty?
+    if !Setting.registration_period?
+      false
+    elsif submissions.empty?
```

Comment:
But actually why here we are checking the presence of submissions and in the invitations_reminder not?